doc/source/downloading_cesm.rst

```diff
@@ -120,50 +107,51 @@ columns of output, as in this example:
        ./components/rtm
        ./components/ww3
 
+You should now have a complete copy of the CESM2 source code in your /path/to/my_cesm_sandbox. 
+
 If there were problems obtaining an external, you might instead see something like:
 
 .. code-block:: console
 
    e-  ./components/cam
 
-For more information, run:
+This might happen if there was an unexpected interruption while downloading. In 
```

Comment:
Edit for grammar: We shouldn't have a sentence beginning "In which case". I'd just remove the "In which case" here, so the sentence begins, "First try rerunning".

Reply:
fixed.